cmd/zbpack/main_test.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+)
+
+var binName = "zbpack"
+
+func TestMain(m *testing.M) {
+	fmt.Println("Running build...")
+
+	if runtime.GOOS == "windows" {
+		binName += ".exe"
+	}
+
+	build := exec.Command("go", "build", "-o", binName)
+
+	if err := build.Run(); err != nil {
+		_, _ = fmt.Fprintf(os.Stderr, "Cannot build tool %s: %s", binName, err)
+		os.Exit(1)
+	}
+
+	fmt.Println("Running test...")
+	result := m.Run()
+
+	fmt.Println("Cleaning up")
+	_ = os.Remove(binName)
+
+	os.Exit(result)
+}
+
+func TestCLI(t *testing.T) {
+
+	dir, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cmdPath := filepath.Join(dir, binName)
+
+	t.Run("show help information", func(t *testing.T) {
+		cmd := exec.Command(cmdPath, "-h")
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		if !strings.Contains(string(out), "Usage") {
+			t.Fatal("expected help output, but got: ", string(out))
+		}
+	})
+
+	t.Run("show error when not give path", func(t *testing.T) {
+		cmd := exec.Command(cmdPath)
+		out, err := cmd.CombinedOutput()
+		if err == nil {
+			t.Fatal("expected error, but got nil")
+		}
+
+		if !strings.Contains(string(out), "Error") {
+			t.Fatal("expected error output, but got: ", string(out))
+		}
+	})
+
+	t.Run("only show info when give --info flag", func(t *testing.T) {
+		testFilePath := "../../tests/node-corepack"
```

Comment:
It should be an absolute path with `cmd/zbpack` as base to prevent unexpected result.